repour/server/endpoint/external_to_internal.py

```diff
@@ -39,15 +43,11 @@ async def translate_external_to_internal(external_git_url):
     elif not gerrit_server.endswith("/"):
         gerrit_server = gerrit_server + "/"
 
-    is_scp_like = external_git_url.startswith("git@")
+    is_scp_like = re.match(SCP_LIKE_URL_REGEX, external_git_url)
 
-    if is_scp_like:
```

Comment:
wait, why are you deleting this stuff

Reply:
Cause 'git@' is not signature of scp_like url so I cannot split by '@' anymore. So instead the scheme is going to be set manually when the url starts with 'git@'

Reply:
Technically its not scheme but the user..so it's kind of workaround.